coop_membership/models/ir_module_module.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+from odoo import api, models
+
+
+class IrModuleModule(models.Model):
+    _inherit = 'ir.module.module'
+
+    @api.model
+    def _update_coop_memberhsip_partner_form(self):
+        """
+        Fix coop_membership remove sex field some view are not update,
+        can be remove this method when migrate to version 13.
+        """
+        view_res_partner_form = self.env.ref(
+            'coop_membership.view_res_partner_form',
+            raise_if_not_found=False
+        )
+        if view_res_partner_form:
+            view_res_partner_form.unlink()
+        view_bdm_search = self.env.ref(
+            'coop_membership.view_bdm_search',
+            raise_if_not_found=False
+        )
+        if view_bdm_search:
+            view_bdm_search.unlink()
+        view_res_partner_form_membership = self.env.ref(
+            'coop_membership.view_res_partner_form_membership',
+            raise_if_not_found=False
+        )
+        if view_res_partner_form_membership:
+            view_res_partner_form_membership.unlink()
+        view_res_partner_form_membership_manager = self.env.ref(
+            'coop_membership.view_res_partner_form_membership_manager',
+            raise_if_not_found=False
+        )
+        if view_res_partner_form_membership_manager:
+            view_res_partner_form_membership_manager.unlink()
```

Comment:
Can we do this in a pre-migration script instead?

They're easier to maintain, and it's clearer that they should be removed in future versions

Reply:
Also, now that we no longer need the `sex` field, you can also remove it from the .py file